### ISSUE TITLE
Alias request: Microsoft.Storage/storageAccounts/isHnsEnabled

### ISSUE THREAD

**Issue by kethiren** [Azure/azure-policy#281]
### Scenario

Trying to deny DLS Gen 2, but still allow Storage Accounts to be created.

### Alias

Need property 'isHnsEnabled' availble as a name space the same way as 'accessTier' and 'supportsHttpsTrafficOnly':
Microsoft.Storage/storageAccounts/accessTier; 
Microsoft.Storage/storageAccounts/supportsHttpsTrafficOnly; 

Ref. this part of a template definition for a Storage Account deployment
(...)
            "properties": {
                "accessTier": "[parameters('accessTier')]",
                "supportsHttpsTrafficOnly": "[parameters('supportsHttpsTrafficOnly')]",
                "isHnsEnabled": "[parameters('isHnsEnabled')]"
            }
(...)
In other words this namespace: Microsoft.Storage/storageAccounts/isHnsEnabled

### Example policy definition

{
"policyRule": {
  "if": {
  "allOf": [{
  "field": "type",
    "equals": "Microsoft.Storage/storageAccounts"
  },
  {
  "field": "Microsoft.Storage/storageAccounts/isHnsEnabled",
    "equals": "true"
  }]
  },
  "then": {
    "effect": "deny"
    }
  }
}
